### PR TITLE
Update fund_issue.md

### PR DESCRIPTION
**Pull request by thierryOse**
Stripe API exploration

### CODE REVIEW

File: architechture/fund_issue.md [hunk 1]
```diff
@@ -1,60 +1,44 @@
-```mermaid
```
Comment (LaurianeOSE) [inline architechture/fund_issue.md:1]: Please not delete this part

File: architechture/fund_issue.md [hunk 2]
```diff
@@ -1,60 +1,44 @@
...
     SmartContract->>SmartContract : mint(LINAmount)
     SmartContract->>SmartContract : fund(amount=LINAmount, what=Linux_issue_1)
     
-    SmartContract-->>Gateway: ok
-    deactivate SmartContract
-
-    deactivate Gateway
-```
-
-### DESCRIPTION
```
Comment (LaurianeOSE) [inline architechture/fund_issue.md:49]: Please not delete this part

File: architechture/fund_issue.md [hunk 3]
```diff
@@ -1,60 +1,44 @@
...
+    Note over OSE_backend: Store transaction
+    Note over OSE_backend: Trigger Binance process
+    Note over Binance: Binance account has been previously <br/>funded with a bag of $
+    OSE_backend->>Binance: swap(amount=100$, to=USDC)
+    Binance-->>OSE_backend: USDCAmount
 
     note over CompanyWallet: Company wallet has been previously <br/> funded with a bag of USDC
-    Gateway->>SmartContract: swap(amount=USDCAmount, to=LIN, signer=CompanyWallet)
-    activate SmartContract
+    OSE_backend->>SmartContract: swap(amount=USDCAmount, to=LIN, signer=CompanyWallet)
```
Comment (LaurianeOSE) [inline architechture/fund_issue.md:38]: mermaid is camelcase
not "OSE_backend" but "OSEBackend"

**Comment by thierryOse** [comment]
```mermaid
sequenceDiagram
    title: Fund issue

    actor Client

    participant OSEFrontend
    participant Stripe
    participant CompanyBankAccount
    participant OSEBackend

    participant Binance
    participant SmartContract
    participant CompanyWallet
    
    
    Note over Client: The client wants to fund the <br/>issue number 1 of Linux with 100$

    Client->>OSEFrontend: login
    
    Note over OSEFrontend: PAY button <br/> embedded stripe.Checkout method
    OSEFrontend->>Stripe: pay(amount=100, <br/> currency=USD, <br/> note=Linux_issue_1)

    Note over Stripe: verify and confirm payment
    Stripe->>CompanyBankAccount: 
    CompanyBankAccount-->>Stripe: 
    Note over Stripe: Webhook payment status
    Stripe-->>OSEFrontend: OK / NOK

    Note over OSEFrontend: display success or error page <br/> reconcile with metadata (payment details)
    OSEFrontend->>OSEBackend: sendPayment(amount=100$, <br/> from=Client, note=Linux_issue_1)
    Note over OSEBackend: Store transaction
    Note over OSEBackend: Trigger Binance process
    Note over Binance: Binance account has been previously <br/>funded with a bag of $
    OSEBackend->>Binance: swap(amount=100$, to=USDC)
    Binance-->>OSEBackend: USDCAmount

    note over CompanyWallet: Company wallet has been previously <br/> funded with a bag of USDC
    OSEBackend->>SmartContract: swap(amount=USDCAmount, to=LIN, signer=CompanyWallet)
   
    CompanyWallet->>SmartContract: send(amount=USDCAmount)
    SmartContract->>SmartContract : mint(LINAmount)
    SmartContract->>SmartContract : fund(amount=LINAmount, what=Linux_issue_1)
    
    SmartContract-->>OSEBackend: ok

```

### DESCRIPTION
* `SmartContract` is a contract that aggregate two other smart contracts: the Augmented Bonding Curve for `OSE` and `LIN` token, as described in the white paper. This contract is used to mint new tokens and fund an issue.
More details on diagram: TODO

### FUTURE IMPROVEMENTS

> Be sure that the swap is processed even if the gateway crashes. 

> We should add other exchanges to have a back-up if Binance is down.

### VIDEO
[Explanatory video](https://youtu.be/CCO8v1yHzKA) of this diagram.

**Pull request closed without merging**